src/pages/_app.tsx

```diff
@@ -1,17 +1,15 @@
 import Navbar from "../components/Navbar";
 import { Outlet, Route, Router, Routes } from "react-router-dom";
-import EditProjectPage, { CreateProjectPage } from "./projects/_components/[projectId]";
+import EditProjectPage from "./projects/[projectId]";
 
 function App() {
 
   return (
     <>
+    <div className="container max-w-screen-xl mx-auto px-5">
```

Comment:
No need to have a Fragment her just use the div component directly

Reply:
fixed